extras/wificfg/wificfg.c

```diff
@@ -1819,7 +1855,7 @@ void wificfg_init(uint32_t port, const wificfg_dispatch *dispatch)
     /* Validate the configuration. */
 
     if (wifi_sta_enable && (!wifi_sta_ssid || !wifi_sta_password ||
-                            strlen(wifi_sta_ssid) < 1 ||
+                            strlen(wifi_sta_ssid) < 8 ||
```

Comment:
I don't think there is such a limit on the ssid length.

Reply:
I will revisit this one as I ran into the insecure ESP_xxxxxx AP being started (instead of the secured one with the selected name) when using a short SSID.